src/frontend/sharedComponents/Editor/index.tsx

```diff
@@ -144,10 +144,9 @@ export const Editor = ({
                 }
                 // observationId must exist if there is a saved photo
                 if (isSavedPhoto(att)) {
```

Comment:
For now I think remove the check below, and just do this the same as for audio (before this change)

```suggestion
                if (isSavedPhoto(att) && observationId) {
```